doc/next-steps/customizing-your-list.md

```diff
@@ -0,0 +1,314 @@
+
+  - ```tsx
+      {
+        column: {
+          sortable, // Boolean that reflects whether or not sorting has been applied to your column
+          key, // Unique key for your column
+          label, // The column's label
+        },
+        meta, // Contains metadata, including sort request pending status
+        sortBy, // The field that your list is sorted by. This will always equal the field that your column corresponds to, if applicable
+        direction, // The direction that your list is currently sorted by
+        setSort, // A function for updating the list's sort
+      }
+    ```
+
+This example is a little contrived because the `label` we're destructuring will always be equal to the `label` prop we set on our `Column`, so we could have simply omitted the `label` prop entirely and hard-coded the value in the return of our callback. If you _don't_ set a `label` prop on your `Column`, `label` will equal the column's default label.
+
+- Just like `renderDataValue`, `renderHeaderValue` can be replaced with `HeaderValueComponent`, which works similarly to `DataValueComponent`.
+
+- The `sortable` prop tells Hatchify whether or not to render the default column sorting UI in the column's header cell.
+
+  ```tsx
+  sortable={true}
+  ```
+
+  It defaults to `true` as long as your `field` prop specifies a valid field on your record, so in this case it's redundant to include it--we just did so for demonstration purposes. An example use case for setting `sortable` to `false` would be if you wanted to handle the sorting UI yourself through the `renderHeaderValue` prop.
+
+Okay! We just learned how to selectively modify an existing column--now let's try adding brand new columns to our list.
+
+### `type="prepend|append"`
+
+#### Code
+
+✏️ Update `/src/App.tsx` to the following:
+
+```tsx
+import { useState } from "react"
+import { hatchifyReact, HatchifyProvider, createJsonapiClient } from "@hatchifyjs/react"
+import { createTheme, ThemeProvider } from "@mui/material"
+import * as Schemas from "../schemas.js"
+
+export const hatchedReact = hatchifyReact(createJsonapiClient("/api", Schemas))
+
+const TodoList = hatchedReact.components.Todo.Collection
+const TodoEmpty = hatchedReact.components.Todo.Empty
+const TodoColumn = hatchedReact.components.Todo.Column
+
+const App: React.FC = () => {
+  return (
+    <ThemeProvider theme={createTheme()}>
+      <HatchifyProvider>
+        <TodoList>
+          <TodoEmpty>
+            <strong>There are no todos. Time to take a break!</strong>
+          </TodoEmpty>
+          <TodoColumn type="replace" field="name" label="ToDo" renderDataValue={({ value }) => <strong>{value}</strong>} renderHeaderValue={({ column: { label } }) => <strong>{label} Items</strong>} sortable={true} />
+          {/* 👀 */}
+          <TodoColumn type="append" label="Actions" renderDataValue={({ record }) => <button onClick={() => alert(`${record.id}`)}>View id</button>} />
```

Comment:
maybe put this before the replace column just to hammer the fact that `append` takes precedence in ordering over JSX

Reply:
Done